src/core/loopback-setup.c

```diff
@@ -26,6 +26,8 @@
 #include "missing.h"
 #include "netlink-util.h"
 
+#define LOOPBACK_SETUP_TIMEOUT_USEC (5 * USEC_PER_SEC)
```

Comment:
the commit msg is a bit misleading, as the kernel will add the addresses automatiucally, but it won't set the iface IFF_UP, that's something we really have to do, and the wait call in the loop waits for that too.